diagnostic.js

```diff
@@ -29,26 +29,72 @@ mongoose.Promise = global.Promise;
 mongoose.connect('mongodb://localhost/mongoose-crud');
 
 /// ADD YOUR CODE BELOW
+const db = mongoose.connection;
+const Movement = require('./models/movement.js')
+const done = function() {  db.close(); };  //close connection
```

Comment:
`db`  and `done` are not necessary here - they was part of the UI functionality of the command line app. We needed to ensure that the Mongo connection was closed each time the script finished running so that there weren't lots of open connections at the end of the day. Here, we (almost certainly) want to keep the connection open.